src/components/StaticContent/Partners.re

```diff
@@ -97,7 +97,7 @@ module OrgDetails = {
               ]),
             ])
           )
-          src={Animal.cdnBase ++ logo}
+          src={CONSTANTS.cdnBase ++ logo}
```

Comment:
Lots of these changes, I just moved the cdn string to the `CONSTANTS` file, makes more sense there!